src/pages/kwonojae/Main/components/writecomment.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSmile } from '@fortawesome/free-regular-svg-icons';
+import Comment from './comment';
+import './writecomment.scss';
+
+export default class Writecomment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      currentCommentInputData: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/kwonojae/commentData.json')
+      .then(response => response.json())
+      .then(commentData => {
+        this.setState({ commentList: commentData });
+      });
+  }
+
+  currentCommentInput = e => {
+    this.setState({ currentCommentInputData: e.target.value });
+  };
+
+  commentPost = e => {
+    const { commentList, currentCommentInputData } = this.state;
+    e.preventDefault();
+
+    if (currentCommentInputData) {
+      this.setState({
+        commentList: [
+          ...commentList,
+          { id: commentList.length + 1, value: currentCommentInputData },
```

Comment:
length값을 활용하면 만약 댓글이 지워졌다가 다시 생성되고 할 경우에 id값이 겹쳐서 문제가 생길 여지가 있어보이네요 다른 방법을 한번 찾아보세요

Reply:
새로 생성되는 댓글의 id = 가장 최신 댓글의 id +1 로 수정했습니다. 단 제가 생각한 구현 방법에서 기존에 달린 댓글이 없을때 첫 댓글의 id 가 정상적으로 부여되질않아서 조건문을 써봤습니다.
```suggestion
          {
            id: commentList.length
              ? commentList[commentList.length - 1].id + 1
              : 1,
            value: nowCommentInputData,
          },
```